dashboard/db/migrate/20170224150102_remove_activity_id_from_gallery_activities.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveActivityIdFromGalleryActivities < ActiveRecord::Migration[5.0]
+  def change
```

Comment:
Hah, I knew I was feeling some deja vu when I made this comment.

I guess I just like the comfort of having reversible commits actually be reversible without needing additional knowledge. It's not a huge issue, though.